tests/_utils/wait_process_exit

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# parameter 1: process name
+
+process=$1
+
+while true
+do
+    pgrep $process > /dev/null 2>&1
+    ret=$?
+    if [ "$ret" != "0" ]; then
+        echo "process $process already exit"
+        break
+    fi
+    sleep 0.1
+    echo "wait process $process exit..."
```

Comment:
`echo` every `0.1` second, too much log?

Reply:
changed to `0.2s`, not a lot of logs, processes often exit within 1s in these integration test